src/Pages/logIn.cshtml.cs

```diff
@@ -56,8 +56,8 @@ public async Task<IActionResult> OnPost(string returnUrl)
         public IActionResult OnPostForgotPassword()
         {
             if (!ModelState.IsValid) return Page();
-
-            Response.Cookies.Append("Email", Email);
+            var option = new CookieOptions() { Expires = DateTime.Now.AddMinutes(15) };
```

Comment:
Just wondering, how does this expiration work? In a little more detail, please

Reply:
It just an option you can add, how the cookies are handled in the deep back I don't really know :D
I guess it just has a date, from which it is deleted.